### PR TITLE
fix laser engraver consuming the lens

### PR DESCRIPTION
**Pull request by Fiyorin**
the lens isn't marked "NC" and gets consumed in the crafting
this fixes it

image

### CODE REVIEW

File: src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java [hunk 1]
```diff
@@ -255,28 +255,28 @@ public void run() {
         // Optical to Wafer Buff Recipes
         GT_Values.RA.addLaserEngraverRecipe(
                 ItemList.Circuit_Silicon_Wafer6.get(1L),
-                GT_OreDictUnificator.get(OrePrefixes.lens, Materials.NetherStar, 1L).copy(),
+                GT_OreDictUnificator.get(OrePrefixes.lens, Materials.NetherStar, 1L).copy().splitStack(0),
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: why not set 1L to 0L ?
like in this recipe https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/600/files#diff-99457ece25a9877f6a71a1e17dffcebe33cc56c5d36d69b707b515bfbce3edcfL219

Comment (Fiyorin) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: i don't know, @TheEpicGamer274  wrote the code

Comment (minecraft7771) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: Use 0L as dream suggested.

Comment (chochem) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: GT_OreDictUnificator would just give null

Comment (chochem) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: you could use GT_Utility.copyAmount(0, GT_OreDictUnificator.get(OrePrefixes.lens, Materials.NetherStar, 1)) though to keep it consistent with the rest of the file

Comment (minecraft7771) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: Hmm, we should change that possibly or what do you think about it?

Comment (TheEpicGamer274) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: I agree with chochem

**Comment by Fiyorin** [comment]
> sorry about the kerfuffle bois

no problem, this 100x better than to continue using the old recipe

File: src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java (hunk 1 above)
Comment (Fiyorin) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: ye i'm testing it with chochems idea

Comment (Fiyorin) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: it would just crash if i gave it 0L

Comment (TheEpicGamer274) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: sounds about right

Comment (TheEpicGamer274) [reply src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java:258]: I believe my water line recipes have the exact thing for lens use that is the best to use usually

**Review by chochem — APPROVED** [review]

**Pull request merged**